moveit_commander/src/moveit_commander/move_group.py

```diff
@@ -211,17 +200,16 @@ def set_joint_value_target(self, arg1, arg2=None, arg3=None):
         Instead, one IK solution will be computed first, and that will be sent to the planner.
         """
         if isinstance(arg1, RobotState):
-            if not self._g.set_state_value_target(conversions.msg_to_string(arg1)):
+            raise NotImplementedError("set_state_value_target is not implemented")
```

Comment:
Any thoughts?

Reply:
pybind11 allows for overloads. Why not provide a method to set joints from RobotState msg?
But, maybe I didn't get the actual problem. If so, please provide a more verbose description :wink:

Reply:
AFAIK this has been broken, as `set_state_value_target` has not been implemented in the wrapper. Guess we can just take `RobotState::joint_state`?